src/clusterfuzz/_internal/platforms/android/adb.py

```diff
@@ -448,6 +448,13 @@ def stop_cuttlefish_device():
         stop_cvd_cmd, timeout=RECOVERY_CMD_TIMEOUT, on_cuttlefish_host=True)
     time.sleep(STOP_CVD_WAIT)
 
+  kill_crosvm = 'pkill crosvm'
```

Comment:
Can you please change these inline constants to class-level constants?

something like:

```adb.py

CMD_KILL_CROSVM = 'pkill crosvm'
...
execute_command(
      CMD_KILL_CROSVM, timeout=RECOVERY_CMD_TIMEOUT, on_cuttlefish_host=True)
```